src/test/FreeC/Pass/ExportPassTests.hs

```diff
@@ -0,0 +1,126 @@
+-- | This module contains tests for "FreeC.Pass.ExportPass".
+
+module FreeC.Pass.ExportPassTests
+  ( testExportPass
+  )
+where
+
+import           Test.Hspec
+
+import           Data.List                      ( find )
+import           Data.Maybe                     ( fromJust )
+import qualified Data.Set                      as Set
+
+import qualified FreeC.Backend.Coq.Syntax      as Coq
+import           FreeC.Environment
+import           FreeC.Environment.Entry
+import           FreeC.Environment.ModuleInterface
+import           FreeC.IR.Syntax               as IR
```

Comment:
Oh, and I've just noticed that you were writing `Qual` and `Ident` instead of `IR.Qual` and `IR.Ident`. Always qualify imports of `Syntax` modules.

```suggestion
import qualified FreeC.IR.Syntax               as IR
```

Reply:
I did that in an earlier commit. I somehow overlooked that.